sdk/messaging/azservicebus/client_test.go

```diff
@@ -471,6 +472,48 @@ func TestNewClientUnitTests(t *testing.T) {
 		require.EqualValues(t, ns.FQDN, "mysb.windows.servicebus.net")
 	})
 
+	t.Run("TracerIsSetUp", func(t *testing.T) {
+		hostName := "fake.servicebus.windows.net"
+		// when tracing provider is not set, use a no-op tracer.
+		client, err := NewClient(hostName, struct{ azcore.TokenCredential }{}, nil)
```

Comment:
Side note, love this construct: "struct{ azcore.TokenCredential }{}". Makes total sense, and I'd never thought of doing it before.

Reply:
copilot 🐐

Reply:
🤖